speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/conformationmenu.cpp

```diff
@@ -0,0 +1,84 @@
+/***************************************************************************
+
+    file                 : savemenu.cpp
+    created              : Tue Mar 18 10:54:12 CET 2022
+    copyright            : (C) 2022 by Maik Vink
+    email                : torcs@free.fr
+    version              : $Id: savemenu.cpp $
+
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include <tgfclient.h>
+
+#include "legacymenu.h"
+
+#include "mediator.h"
+#include "conformationmenu.h"
+#include "mainmenu.h"
+
+static void *MenuHandle = NULL;
+
+static void
+onAcceptExit(void * /* dummy */)
+{
+    SMediator::GetInstance()->SetSaveRaceToDatabase(false);
+    LmRaceEngine().abortRace();  // Do cleanup to get back correct setup files
+    LegacyMenu::self().quit();
+}
+
+static void
+onAcceptRestart(void * /* dummy */)
+{
+    SMediator::GetInstance()->SetSaveRaceToDatabase(false);
+    LmRaceEngine().restartRace();
+}
+
+void *ConformationMenuInit(void *prevMenu, int p_saveWayVersion)
```

Comment:
Fix all spelling mistakes.
```suggestion
void *ConfirmationMenuInit(void *prevMenu, int p_saveWayVersion)
```